hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/state/WrappedWritableKVStateTest.java

```diff
@@ -57,4 +60,61 @@ void commitGoesToDelegateNotBackingStore() {
         assertThat(delegate.get(B_KEY)).isEqualTo(BLACKBERRY); // Has the new value
         assertThat(delegate.get(E_KEY)).isEqualTo(ELDERBERRY); // Has the new value
     }
+
+    @Nested
+    @DisplayName("size")
+    final class SizeTest {
+        /**
+         * Gives size of backing store. Since on WrappedWritableKVState, all the changes are only
+         * buffered to modifications, size of backing store will not change on committing new values.
+         */
+        @Test
+        @DisplayName("Put a key that does not already exist in the backing store")
```

Comment:
Maybe the display name would be clearer with a description also of what should happen. Like "Adding a key that is not in the backing store impacts the size" or something? Not sure if there is a less wordy way of saying it.

Reply:
Yes agreed .. changed names on some other tests as well.